package.json

```diff
@@ -37,10 +37,13 @@
     "git-url-parse": "^6.0.1",
     "jsdom": "^8.3.1",
     "mocha": "^2.4.5",
+    "raw-loader": "^0.5.1",	
     "react": "^0.14.7 || ^15.0.0",
-    "react-addons-test-utils": "^0.14.7 || ^15.0.0",
+    "react-addons-pure-render-mixin": "^15.3.0",
+	"react-addons-test-utils": "^0.14.7 || ^15.0.0",
     "react-dom": "^0.14.7 || ^15.0.0",
-    "sinon": "^1.17.3"
+    "sinon": "^1.17.3",    
+    "style-loader": "^0.13.1"
```

Comment:
So since you are not using any of those new modules for the feature itself you don't need to make any changes to package.json right? I think all those new modules you added are ones required by the earlier example.